packages/customize-widgets/src/controls/inserter-outer-section.js

```diff
@@ -4,117 +4,118 @@
 import { ESCAPE } from '@wordpress/keycodes';
 import { focus } from '@wordpress/dom';
 
-const {
-	wp: { customize },
-} = window;
+export default function getInserterOuterSection() {
```

Comment:
Should we use `build` instead of `get`? `get` to me implies that it's a simple property access. I suppose it doesn't matter 😀